File: unified-prototype/unified-plugin/plugin-common/src/testFixtures/resources/templates/java-library/src/main/java/com/example/lib/Library.java
package com.example.lib;

import com.google.common.collect.ImmutableList;

public class Library {
    public Iterable<String> getMessages() {
        // Verify that Guava is available
        ImmutableList.Builder<String> builder = ImmutableList.builder();
        builder.add("Hello from Java " + System.getProperty("java.version"));

        return builder.build();
    }
}

File: unified-prototype/testbed-kotlin-jvm-library/src/main/kotlin/com/example/lib/Library.kt
package com.example.lib

import com.example.utils.Utils
import com.google.common.collect.ImmutableList

class Library {
    val messages: Iterable<String>
        get() {
            // Verify that Guava is available
            val builder = ImmutableList.builder<String>()
            builder.add("Hello from Kotlin ${KotlinVersion.CURRENT} on JVM ${System.getProperty("java.version")}")

            // Verify that the Kotlin JVM library is available
            val utils = Utils()
            builder.add(utils.welcome)

            return builder.build()
        }
}